hermes-api/src/main/java/pl/allegro/tech/hermes/api/EndpointAddress.java

```diff
@@ -17,28 +17,14 @@
 @JsonSerialize(using = EndpointAddressSerializer.class)
 public class EndpointAddress {
 
-    private static final String ANONYMIZED_PASSWORD = "*****";
```

Comment:
we can't lose backwards compatibility - we need to support both url-like credentials and this separate sturcture you created